### PR TITLE
feat(optimus): [RND-162272] Added OptimusTooltip

### PR DESCRIPTION
**Pull request by witwash**
#### Summary

- Added the `OptimusTooltip` component.
- Added the `OptimusTooltip` story.

<details><summary>Screenshots</summary>

CleanShot 2023-03-15 at 14 40 25

</details>

#### Testing steps

- Check the tooltip story and try out exposed options.

#### Follow-up issues

- [RND-165619] - whole PR was getting out of hand so I've split it into two parts. The second part will contain the `OptimusTooltipWrapper` that would define the behavior and position of the `OptimusTooltip`.

#### Check during review

- Verify against YouTrack issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).

### CODE REVIEW

**Comment by witwash** [comment]
@ookami-kb,
Can you please look at this one? I have a follow-up ready and waiting 🙏

**Comment by witwash** [comment]
@ookami-kb,
ping

File: optimus/lib/src/tooltip/tooltip_arrow.dart [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+import 'package:optimus/optimus.dart';
+
+class TooltipArrow extends StatelessWidget {
```
Comment (ookami-kb) [inline optimus/lib/src/tooltip/tooltip_arrow.dart:4]: If the arrow is drawn separately, there's a thin line between arrow and the main shape, it's visible even on screenshots. I would say, the whole shape should be drawn in custom painter.

**Review by ookami-kb — CHANGES_REQUESTED** [review]

**Comment by witwash** [comment]
@ookami-kb, 
fixed. The new look: 
<details><summary>Preview</summary>

CleanShot 2023-03-15 at 14 40 25


</details>

**Comment by witwash** [comment]
@ookami-kb, ping

**Review by ookami-kb — APPROVED** [review]

**Pull request merged**